pkg/skaffold/util/util.go

```diff
@@ -154,6 +154,68 @@ func ReadConfiguration(filename string) ([]byte, error) {
 	}
 }
 
+// A type holding configuration properties
+type ConfigProperties map[string]string
+
+// Tests if a file exists
+func FileExists(filapath string) bool {
+	if _, err := os.Stat(filapath); !os.IsNotExist(err) {
+		if err == nil {
+			return true
+		}
+	}
+	return false
+}
+
+// Reads an environment file  that contains a set of environment variables
+func ReadEnvironmentConfigProperties(pathToSkaffoldFile string) (*viper.Viper, error) {
+	directory := filepath.Dir(pathToSkaffoldFile)
+	baseName := filepath.Base(pathToSkaffoldFile)
+	targetDirectory := ""
+	if baseName != "skaffold.yaml" && baseName != "skaffold.yml" {
+		targetDirectory = pathToSkaffoldFile
+	} else {
+		targetDirectory = directory
+	}
+	logrus.Debugf("Will load environment file from :%s", targetDirectory)
+	ret := viper.New()
+	ret.SetConfigType("yaml")
+	ret.SetConfigName("env")
```

Comment:
I personally want something that changes this to something else, like `skaffold --env prod.yaml`

Reply:
+1, this should be configurable through a CLI flag